<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChecksumConfigDialog</class>
 <widget class="QDialog" name="ChecksumConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckCode Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Algorithm:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbAlgorithm"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>From frame byte:</string>
       </property>
       <property name="toolTip">
        <string>Start byte position from entire frame (includes frame header)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spStartByte">
       <property name="minimum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>To frame byte (inclusive):</string>
       </property>
       <property name="toolTip">
        <string>End byte position from entire frame (includes frame header)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spEndByte">
       <property name="minimum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_endian">
       <property name="text">
        <string>Byte order:</string>
       </property>
       <property name="toolTip">
        <string>Byte order for multi-byte checkCode comparison</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbEndianness">
       <item>
        <property name="text">
         <string>Little Endian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Big Endian</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Info:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lInfo">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChecksumConfigDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
